requirements.txt

```diff
@@ -1,3 +1,5 @@
 ansible==2.5.9
```

Comment:
Can you remove this Ansible requirement? It was put here in error a week or so ago.

Reply:
removing ...